examples/cpp/10_inheritance/src/main.cpp

```diff
@@ -71,17 +69,17 @@ int main(int argc, char *argv[]) {
 
     /* Create entities which share the Mass component from a base */
     ecs.entity("MyInstance1")
-        .add_instanceof(LightEntity)
+        .add(flecs::IsA, LightEntity)
```

Comment:
Would love for some better naming, like a method .instantiate(LightEntity), love the removal of add_childof though!

Reply:
I want to reduce the API footprint, so I think I'll stick to this :) Might take a bit to get used to, but in the context of the new relationship/query framework this makes it clearer how instancing is implemented